user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -1845,7 +1900,7 @@ define([
   function customSort(response) {
 
     var sortFunction = function (a, b) {
-      return window.parent.localeCompare(a.file.title, b.file.title);
+     return window.parent.localeCompare(a.file.name, b.file.name);
```

Comment:
Actually, sorting should be by `title`, when not non-empty. If not present, would default to using `name`. No?

Reply:
ah, yes, in the new model, most files didn't have a title, but they all had a name, so I opted for name. I will change it to use title if it is present.

Reply:
Both roots and VFS files/folders.